src/runtime/math.py

```diff
@@ -215,25 +227,76 @@ def fabs(x: f64) -> f64:
         return -x
     return x
 
+@overload
+def fabs(x: i64) -> i64:
```

Comment:
```suggestion
def fabs(x: i64) -> f64:
```

Shoudln't we change the return type to `f64`.

Reply:
Please see #599. I was getting errors when doing this.

Reply:
You should return `float(x)` instead of `x`. Since you are returning an integer from a function which is supposed to return a floating point, the error in #599 is bound to happen.